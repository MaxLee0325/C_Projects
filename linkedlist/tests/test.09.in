new
append alpha
append beta
append gamma
append delta
iterator
hasnext
next
add epsilon
next 
set eta
next 
remove
next
hasnext
end
