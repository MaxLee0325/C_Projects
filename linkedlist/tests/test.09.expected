[ ]
[ alpha ]
[ alpha beta ]
[ alpha beta gamma ]
[ alpha beta gamma delta ]
[ alpha beta gamma delta ]
Has next
[ alpha beta gamma delta ]
Next: alpha
[ alpha beta gamma delta ]
Added: 'epsilon'
[ alpha epsilon beta gamma delta ]
Next: beta
[ alpha epsilon beta gamma delta ]
Set: 'beta' -> 'eta'
[ alpha epsilon eta gamma delta ]
Next: gamma
[ alpha epsilon eta gamma delta ]
Removed: 'gamma'
[ alpha epsilon eta delta ]
Next: delta
[ alpha epsilon eta delta ]
No next
[ alpha epsilon eta delta ]
